validator.php

```diff
@@ -33,6 +33,11 @@
     return true; // any other file gets checks and any other folder gets iterated
 };
 
+$isAcceptableVersionConstraint = function ($versionString) {
+    return (bool) preg_match('/^(\\<|\\>)(=){0,1}([1-9]\d*)(\.[1-9]\d*)*(-(alpha|beta|rc)[1-9]\d*){0,1}$/', $versionString);
```

Comment:
Looks like we now fail to handle versions just having `0` parts.

This should work:

```
/^(\\<|\\>)(=){0,1}(([1-9]\d*)|0)(\.([1-9]\d*)|0)*(-(alpha|beta|rc)[1-9]\d*){0,1}$/